src/game/objects/traps/movable_block.c

```diff
@@ -289,7 +301,7 @@ bool MovableBlock_TestPush(
         return false;
     }
 
-    if (((int32_t)floor->floor << 8) != y) {
+    if (Room_GetHeight(floor, x, y, z) != y) {
```

Comment:
This - and the similar change in `MovableBlock_TestPull` - fixes a separate bug in TR1, where you can push/pull a block onto a slope once. Could we make this optional? There are some secrets in the randomizer that make use of this glitch (and the other OG glitches, whose fixes have config settings).

Reply:
Yes i think this change can be reverted. It was to try and get stacking working on room portals. But since that is not supported anymore, I don't think it's needed.

Reply:
We can raise another issue for adding an option to fix being able to push onto a slope - it's fixed in TR2 onwards anyway.